CorsixTH/Lua/app.lua

```diff
@@ -313,6 +313,39 @@ function App:init()
   return true
 end
 
+---
+-- Initialises a Lua DBGp client and
+-- connects it to a DBGp server so that
+-- it can debug lua scripts at runtime.
+-- @return error_message
```

Comment:
As long as LDocGen also recognizes and parses function comments 
correctly (to get http://www.armedpineapple.co.uk/cth/docs/) I'm fine 
with either way.

/Edvin

On 2015-01-05 16:10, J-Shep wrote:
>
> In CorsixTH/Lua/app.lua 
> <https://github.com/CorsixTH/CorsixTH/pull/439#discussion-diff-22465101>:
>
> > @@ -313,6 +313,39 @@ function App:init()
> >    return true
> >  end
> >
> > +---
> > +-- Initialises a Lua DBGp client and
> > +-- connects it to a DBGp server so that
> > +-- it can debug lua scripts at runtime.
> > +-- @return error_message
>
> I personally think the LDocGen function comment convention should be 
> changed to be the same as the LuaDoc convention I've used here because 
> a) Eclipse makes these multi-line comments blue where as all other Lua 
> comments are green and b) a CorsixTH programmer could start a new 
> comment line by pressing return without having to type: --!
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/CorsixTH/CorsixTH/pull/439/files#r22465101>.
>